src/Azure.Functions.Cli/Common/DurableManager.cs

```diff
@@ -0,0 +1,294 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Functions.Cli.Interfaces;
+using Colors.Net;
+using DurableTask.AzureStorage;
+using DurableTask.Core;
+using DurableTask.Core.History;
+using Microsoft.Azure.WebJobs.Script;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using static Colors.Net.StringStaticMethods;
+
+namespace Azure.Functions.Cli.Common
+{
+    internal class DurableManager : IDurableManager
+    {
+        private readonly AzureStorageOrchestrationService _orchestrationService;
+
+        private readonly TaskHubClient _client;
+
+        private string TaskHubName;
+
+        private string ConnectionString;
```

Comment:
It would be better to make these `readonly` and to follow existing naming conventions, e.g.
```csharp
private readonly string _taskHubName;
private readonly string _connectionString;
```
And then I think you can do this to assign them.
```csharp
this.SetConnectionStringAndTaskHubName(secretsManager, ref _taskHubName, ref _connectionString);
```